src/Pages/Main/Main.js

```diff
@@ -1,9 +1,47 @@
 import React, { Component } from 'react';
+import MainCategory from './MainCategory';
+import MainDiy from './MainDiy';
+import MainStories from './MainStories';
 import './Main.scss';
 
 class Main extends Component {
+  constructor() {
+    super();
+    this.state = { MainStoriesdata: [], isNavOpen: false };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/mainStoriesData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          MainStoriesdata: data,
+        });
+      });
+  }
+  openNav = () => {
+    this.setState({ isNavOpen: !this.state.isNavOpen });
+  };
```

Comment:
```suggestion
  }
  
  openNav = () => {
    this.setState({ isNavOpen: !this.state.isNavOpen });
  };
  
```
구분되는 로직은 가독성을 위해 한 줄씩 띄워주세요